cmd/config_test.go

```diff
@@ -139,5 +145,185 @@ var _ = Describe("Config", func() {
 				Eventually(session).Should(clitest.ShouldFail())
 			})
 		})
+
+		Describe("validating configs", func() {
+			config := "version: 2.1"
+			BeforeEach(func() {
+				command = exec.Command(pathCLI,
+					"config", "validate",
+					"--skip-update-check",
+					"--token", token,
+					"--host", tempSettings.TestServer.URL(),
+					"-",
+				)
+
+				stdin, err := command.StdinPipe()
+				Expect(err).ToNot(HaveOccurred())
+				_, err = io.WriteString(stdin, config)
+				Expect(err).ToNot(HaveOccurred())
+				stdin.Close()
+			})
+			It("returns an error when validating a config", func() {
+				query := `query ValidateConfig ($config: String!, $pipelineValues: [StringKeyVal!]) {
+					buildConfig(configYaml: $config, pipelineValues: $pipelineValues) {
+						valid,
+						errors { message },
+						sourceYaml,
+						outputYaml
+					}
+				}`
+
+				r := graphql.NewRequest(query)
+				r.Variables["config"] = config
+				r.Variables["pipelineValues"] = pipeline.PrepareForGraphQL(pipeline.FabricatedValues())
```

Comment:
Bit of a nitpick - wdyt about this being a var set with `BeforeEach` as well, since we're doing that for `config` too? Would `FabricatedValues` return something different each time?